js/src/components/MediaUpload.js

```diff
@@ -78,37 +78,59 @@ class MediaUpload extends React.Component {
 	}
 
 	/**
-	 * Renders the output.
+	 * Renders a remove button when a company image is set.
```

Comment:
I would remove 'company' from this comment, because it's an implementation detail.